README.md

```diff
@@ -30,7 +30,10 @@ To do this, simply run `git submodule update --init --recursive` in git bash and
 
 **OpenDream requires .NET 8.** You can check your version by running `dotnet --version`. It should be at least `8.0.0`.
 
-To build, one can use a C# compiler (such as MSBuild) to compile the various projects described in the solution.
+To build, one can use a C# compiler (such as MSBuild) to compile the various projects described in the solution. To us MSBuild, simply run `dotnet build` in the OpenDream directory. This will build all of the solutions and put the resultant binaries in `bin`
```

Comment:
```suggestion
To build, one can use a C# compiler (such as MSBuild) to compile the various projects described in the solution. To use the .NET build system, simply run `dotnet build` in the OpenDream directory. This will build all of the solutions and put the resultant binaries in `bin`
```